recipes/seqlogo/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "seqLogo" %}
+{% set version = "0.0.6" %}
+{% set sha256 = "606d6e963debe4ed5dc8b2d490e00de87f05a577f07e01329398bc9028e7f9d1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True # [win32]
+  skip: True # [win64]
+  skip: True # [py<34]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - numpy
+    - pandas
+    - pip
+    - python
+    - weblogo
+  run:
+    - numpy
+    - pandas
+    - python
+    - weblogo
+
+test:
+  imports:
+    - seqLogo
```

Comment:
Are there any commands you can run that go past importing `seqLogo`?

Reply:
The tests are there to ensure that the package is usable. They shouldn't take a lot of time, but they should trigger the imports (pandas, numpy). That way we don't upload a broken package once this gets merged, and prevent the same when doing updates to the recipe later on.

Reply:
The `imports: [seqLogo]` just leads to a `python -c 'import seqLogo'`, nothing more. That's often not enough to test whether a module is actually usable.

Reply:
Attempting to address this with 2f979e49da4d1a168faea67732ce3db658181e34
This does some very basic functionality testing since this is essentially a wrapper to WebLogo (which is already heavily tested). I am only testing the entry points of my side of things.

Reply:
That's alright. It's just to catch things like broken dependencies (e.g. after updating the package).